spec.md

```diff
@@ -319,43 +327,43 @@ recursive resolver. Consequently, a sender MAY treat a policy as valid for up to
 expiration of a cached policy, and to prevent long-term (either malicious or
-active) denials of service, it is important that senders are able to validate a
+active) denials of service, it is important that senders are able to authenticate a
 new policy retrieved for a recipient domain. There are two supported mechanisms
-for policy validation:
+for policy authentication:
 
 * Web PKI: In this mechanism, indicated by the "webpki" value of the "a" field,
-  the sender fetches a HTTPS resource from the URI indicated. For example,
-  a=webpki:https://example.com/.well-known/smtp-sts/current indicates that the
-  sender should fetch the resource
-  https://example.com/.well-known/smtp-sts/current. In order for the policy to
+  the sender fetches a HTTPS resource from a well known URI. For example,
+  a=webpki indicates that the sender should fetch the resource from
+  https://policy._smtp_sts.example.com/current. In order for the policy to
   be valid, the HTTP response body served at this resource MUST exactly match
   the policy initially loaded via the DNS TXT method, and MUST be served from an
   HTTPS endpoint at the domain matching that of the recipient domain.  (As this
-  RFC progress, the authors intend to register .well-known/smtp-sts.  See
-  [@!RFC5785]. See _Future_ _Work_ for more information.)
-
+  RFC progress, the authors intend to register _smtp-sts.  See
+  [@!RFC5785]. See _Future_ _Work_ for more information.). Since the policy is
+  served from a special subdomain, MTA operators may host a separate web server
+  different from their main domain. This is desired when the main domain is operated
+  by a different team. This model also enable a 3rd party (eg. mail service provider) 
+  to host the policy behalf of their business users.
+  
 * DNSSEC: In this mechanism, indicated by the "dnssec" value of the "a" field,
   the sender MUST retrieve the policy via a DNSSEC signed response for the
   _smtp_sts TXT record.
 
-When fetching a new policy when one is not already known, or when fetching a
-policy for a domain with an expired policy, unauthenticated policies MUST be
-trusted and honored. When fetching a policy and authenticating it, as described
-in detail in _Policy_ _Application_, policies will be authenticated using the
-mechanism specified by the existing cached policy.
-
-Note, however, as described in detail in _Policy_ _Application_, that new
-policies MUST NOT be considered as valid if they do not validate on first
-application. That is, a freshly fetched (and unused) policy that has not
-successfully been applied MUST be disregarded.
+When fetching a new policy, or when fetching a policy for a domain with an 
+expired policy or if the sender detects a policy update (by checking policy id
+or computing SHA256 of policy), the new policy SHOULD be authenticated before use. 
+If the policy domain is DNSSEC enabled, then sender MAY skip explicit policy
```

Comment:
If the policy is served over DNSSEC, then there is no need to further authenticate the policy. Yes, it should be either dnssec OR webpki.